pkg/tools/benchmark/job/service/remove.go

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright (C) 2019-2022 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package service manages the main logic of benchmark job.
+package service
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/vdaas/vald/apis/grpc/v1/payload"
+	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/log"
+	"github.com/vdaas/vald/internal/net/grpc/codes"
+	"github.com/vdaas/vald/internal/net/grpc/status"
+)
+
+func (j *job) remove(ctx context.Context, ech chan error) error {
+	log.Info("[benchmark job] Start benchmarking remove")
+	if j.removeConfig == nil {
```

Comment:
How about the implementation of checking for nil in the `job.New` function?


https://github.com/vdaas/vald/blob/feature/pkg-internal/impl-benchmark-jobs/pkg/tools/benchmark/job/service/job.go#L113-L131